templates/header.tpl

```diff
@@ -36,6 +35,11 @@
 				<!-- IMPORT partials/menu.tpl -->
 			</div>
 		</nav>
+		<script>
+			const rect = document.getElementById('header-menu').getBoundingClientRect();
+			const offset = Math.max(0, rect.bottom);
+			document.documentElement.style.setProperty('--panel-offset', offset + `px`);
+		</script>
```

Comment:
I am _technically_ not a fan of standalone script blocks, **but in this case** it is necessary for the offset code to function without that viewport shifting on initial page load.

Can we compromise and have an HTML comment added here to refer to `updatePanelOffset();` in `public/persona.js`?